Makefile

```diff
@@ -137,14 +140,21 @@ update: | update-common
 $(DOTHERSIDE): | deps
 	echo -e $(BUILD_MSG) "DOtherSide"
 	+ cd vendor/DOtherSide && \
+		rm -f CMakeCache.txt && \
```

Comment:
This one is created in the "build" directory, so that's where it needs to be pre-emptively deleted.

Reply:
Ah yeah, forgot about that.